src/main/webapp/js/omniComment.js

```diff
@@ -223,19 +223,19 @@ $(document).ready(function() {
         // if not all list elements are hidden within fbResponse, then show fbResponse
         if ($(comment).prop('class').toString().includes(classNameForCommentsInFeedbackResponse)) {
             if ($(comment).parent().find('li[style*="display: none"]').length !== $(comment).parent().find('li').length) {
-                var commentListRegionForFeedbackResponse = $(comment).parent().parent().parent();
+                var commentListRegionForFeedbackResponse = $(comment).closest('tr');
```

Comment:
Unrelated to your changes - this whole portion seems to need a good revamp some day, the comment object is constantly being rewrapped into a jQuery object unnecessarily (once is enough) and the code below reminds me of what we are fixing in this PR ```.parent().parent()...``` and ```.prev().prev().prev()...``` :D